Frameworks/include/Platform/EbrPlatform.h

```diff
@@ -83,8 +83,6 @@ SB_EXPORT double EbrGetMediaTime();
 SB_EXPORT int EbrGetWantedOrientation();
 
 SB_EXPORT const wchar_t* IwGetWritableFolder();
-SB_EXPORT void IwSetWritableFolder(const wchar_t* folder);
```

Comment:
I'd personally prefer to see all the Iw Writable Folder changes in a separate pull request with a separate commit (instead of squashed all together at the end). It will help future bisecting if we regress something.

In fact, I think all framework changes should be separate from this _testing-related_ PR.